pkg/network/setup/netpod/netpod.go

```diff
@@ -548,7 +548,7 @@ func filterSupportedBindingNetworks(specNetworks []v1.Network, specInterfaces []
 			return nil, fmt.Errorf("no iface matching with network %s", network.Name)
 		}
 
-		if iface.Binding != nil || iface.SRIOV != nil || iface.Macvtap != nil {
+		if iface.SRIOV != nil || iface.Macvtap != nil {
```

Comment:
Does this change affect how interfaces whose specify binding are configured by the NetPod/nmstate code?
(Asking just to to be sure)

Reply:
No, the Netpod spec composing skips binding plugin. It affects only the discover code.